src/caret_analyze/architecture/architecture_exporter.py

```diff
@@ -109,24 +109,31 @@ def _timer_cb_to_dict(
         self,
         timer_callback: TimerCallbackStructValue
     ) -> Dict:
-        return  \
-            {
+        d = {
                 'callback_name': timer_callback.callback_name,
                 'callback_type': str(CallbackType.TIMER),
                 'period_ns': timer_callback.period_ns,
                 'symbol': timer_callback.symbol,
             }
+        if timer_callback.construction_order > 0:
+            d['construction_order'] = timer_callback.construction_order
+        return d
 
     def _sub_cb_to_dict(
         self,
         subscription_callback: SubscriptionCallbackStructValue
     ) -> Dict:
-        return {
+        d: Dict[str, Union[str, int]]
+        d = {
             'callback_name': subscription_callback.callback_name,
             'callback_type': str(CallbackType.SUBSCRIPTION),
             'topic_name': subscription_callback.subscribe_topic_name,
             'symbol': subscription_callback.symbol,
         }
+        if subscription_callback.construction_order > 0:
```

Comment:
```suggestion
        if subscription_callback.construction_order and subscription_callback.construction_order > 0:
```

Reply:
fixed in :[8ceaf80](https://github.com/tier4/CARET_analyze/pull/225/commits/8ceaf80c437b7b8155618eeae5e168b34c10bbc7)